backend/src/app.ts

```diff
@@ -69,6 +71,23 @@ server.route('/itemComplete').post(items.itemComplete);
 
 server.route('/email').post(email.send);
 
+const options = {
```

Comment:
Is there anything about testing in the uppy documentation? 
Feels a bit like a black box looking at it.

Reply:
I can't find anything.  I also can't find how to turn off all those console.log on the front-end.